bin/list-all-packages.sh

```diff
@@ -19,6 +19,12 @@ function pkgs() {
             continue
         fi
         echo "${name}-${version}.tar.gz"
+        if [ "${name}" = "kubernetes" ] || [ "${name}" = "k-3-s" ] || [ "${name}" = "rke-2" ]; then
```

Comment:
```suggestion
        if [ "${name}" = "kubernetes" ]; then
```

Reply:
yes, but we dont have 1.18.13 for kubernetes and we do for rke2. I will deduplicate.